src/meshapi/management/commands/scramble_members.py

```diff
@@ -84,28 +84,36 @@ def handle(self, *args: Any, **options: Any) -> None:
         for device in devices:
             device.notes = fake.text()
             _, device.install_date, device.abandon_date = self.fuzz_dates(
-                None, device.install_date, device.abandon_date
+                date.today(), device.install_date, device.abandon_date
             )
             device.save()
 
         print("Scrambling links...")
         links = Link.objects.all()
         for link in links:
             link.notes = fake.text()
-            _, link.install_date, link.abandon_date = self.fuzz_dates(None, link.install_date, link.abandon_date)
+            _, link.install_date, link.abandon_date = self.fuzz_dates(
+                date.today(), link.install_date, link.abandon_date
+            )
             link.save()
 
         print("Scrambling nodes...")
         nodes = Node.objects.all()
         for node in nodes:
             node.notes = fake.text()
-            _, node.install_date, node.abandon_date = self.fuzz_dates(None, node.install_date, node.abandon_date)
+            _, node.install_date, node.abandon_date = self.fuzz_dates(
+                date.today(), node.install_date, node.abandon_date
+            )
             node.save()
 
         print("Done")
 
     @staticmethod
-    def fuzz_dates(request_date: date | None, install_date: date, abandon_date: date) -> Tuple[date | None, date, date]:
+    def fuzz_dates(
+        request_date: date,
+        install_date: Optional[date],
+        abandon_date: Optional[date],
+    ) -> Tuple[date, Optional[date], Optional[date]]:
```

Comment:
Why are these all optional?

Reply:
Because their corresponding attributes are optional on the `Install` object, since those dates don't make sense for installs that haven't yet been installed or abandoned